### DISCUSSION TITLE
How to `mkdir` hidden folder and `writeTextFile` hidden file?

### DISCUSSION THREAD

**Discussion by HuakunShen** [tauri-apps/tauri#11913] [Q&A] [answered]
I am trying to use file system plugin to `mkdir` hidden folder like `.git`, or write hidden files like `.gitignore`.
But got `Unhandled Promise Rejection: forbidden path: /Users/hk/Desktop/.gitignore` error.

I don't see an option in `mkdir` and `writeTextFile`. Is this not possible or there is some capability to configure?

**Comment by FabianLars** [comment, marked as answer]
Does this still happen if you add this to your tauri.conf.json file?
```rs
{
  "plugins": {
    "fs": {
      "requireLiteralLeadingDot": false
    }
  }
}
```

**Reply by HuakunShen** [reply]
Thanks, problem solved!
This should be documented in the documentation.
